spk/jellyfin/Makefile

```
@@ -36,4 +36,18 @@ SERVICE_PORT_TITLE = Jellyfin (HTTP)
 # Admin link
 ADMIN_PORT = $(SERVICE_PORT)
 
+include ../../mk/spksrc.common.mk
+ifeq ($(call version_lt, ${TCVERSION}, 7.0),1)
+# we do not only need the updated libstdc++ library, we also need to
+# adjust the library search path for dotnet to use this version.
+DEPENDS += cross/libstdc++
+POST_STRIP_TARGET = jellyfin_patch_target
+endif
+
 include ../../mk/spksrc.spk.mk
+
+.PHONY: jellyfin_patch_target
+# Set library path to use bundled libstdc++
+jellyfin_patch_target:
+	@$(MSG) "Set library runpath in jellyfin executable."
+	@patchelf --set-rpath /var/packages/$(SPK_NAME)/target/lib
```

you need to specify the target to patch...

```suggestion
	@patchelf --set-rpath /var/packages/$(SPK_NAME)/target/lib $(STAGING_DIR)/share/bin/jellyfin
```

---

Thanks for this